addons/event/static/src/xml/field_icon_selection.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates xml:space="preserve">
+    <t t-name="event.IconSelection">
+        <i t-attt-class="fa #{widget.icon}" t-att-title="widget.title"/>
```

Comment:
Hello, I'm not sure the current state of the PR and I didn't check the rest but I strongly suggest to include the `fa` prefix as par of the icon you store.
That makes it much easier to swap to any other class without having the `fa` class remaining in there.
Also if we ever reconsider upgrading font awesome again, in later version the prefix will become different depending on the style of the icon.

Reply:
Seems legit.